moto/rds2/models.py

```diff
@@ -207,7 +212,7 @@ def to_xml(self):
 
     @property
     def address(self):
-        return "{0}.aaaaaaaaaa.{1}.rds.amazonaws.com".format(self.db_instance_identifier, self.region)
+        return self.address_gen(self.db_instance_identifier, self.region)
```

Comment:
it behaves exactly the same as before, but now we can control the address if necessary (e.g. I want to return the address of a real database)